test.js

```diff
@@ -5,6 +5,7 @@ import m from './';
 
 global.document = jsdom.jsdom();
 global.window = document.defaultView;
+global.requestAnimationFrame = fn => setTimeout(fn, 16);
```

Comment:
rAF tries to match the monitor's refresh rate, which is generally 60hz.

100/60=16.66666666666

Generally speaking a frame is considered 16ms long.